include/bl_storage.h

```diff
@@ -238,6 +290,87 @@ NRFX_STATIC_INLINE void read_implementation_id_from_otp(uint8_t *buf)
  * This is a temporary solution until TF-M has access to NSIB functions.
  */
 
+#ifdef CONFIG_NRFX_RRAMC
```

Comment:
Most of the added functions are basically internal functions, so I think it's less than ideal to have them in the header file. I guess there's precedent for it in this file though, so maybe not in scope to fix here, but maybe just give it a second thought.